inc/session.class.php

```diff
@@ -625,7 +625,12 @@ static function loadLanguage($forcelang = '', $with_plugins = true) {
          // In this case, we may still want to be able to load translations (for instance for requirements checks).
          \Locale::setDefault($trytoload);
       } else {
-         Toolbox::logWarning('Missing required intl PHP extension');
+         Toolbox::logError('Missing required intl PHP extension');
+      }
+
+      if (!extension_loaded('mbstring')) {
+         Toolbox::logError('Missing required mbstring PHP extension');
+         die(1); //do not continue, will throw an exception
```

Comment:
I wonder if, with recently implemented error handlers, the best way to handle that would be to use the native trigger_error function.
```suggestion
         trigger_error('Missing required intl PHP extension', E_USER_ERROR);
      }

      if (!extension_loaded('mbstring')) {
         trigger_error(Missing required mbstring PHP extension', E_USER_ERROR);
```